### PR TITLE
Fix multinode CI

### PR DESCRIPTION
**Pull request by leshikus**
Subj

### CODE REVIEW

File: .github/workflows/deploy-vagrant.yaml [hunk 1]
```diff
@@ -35,3 +28,8 @@ jobs:
           name: logs
           path: logs
           if-no-files-found: ignore
+
+      - name: Sleep
```
Comment (pbchekin) [inline .github/workflows/deploy-vagrant.yaml:32]: Remove before merging?

File: icl-cluster-profiles/profiles/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+calico_endpoint_to_host_action: ACCEPT
+calico_ipip_mode: Never
+calico_network_backend: bird
+calico_vxlan_mode: Never
+containerd_registries_mirrors:
+- mirrors:
+  - capabilities:
+    - pull
+    - resolve
+    host: https://dockerhubcache.caas.intel.com
```
Comment (pbchekin) [inline icl-cluster-profiles/profiles/ci.yaml:12]: Let's move specific configuration to the runner.

File: icl-cluster-profiles/profiles/ci.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  '
+docker_image_repo: dockerhubcache.caas.intel.com
+download_container: false
+ingress_domain: localtest.me
+kube_pods_subnet: 192.168.128.0/18
+kube_service_addresses: 192.168.64.0/18
+ntp_enabled: true
+ntp_manage_config: true
+ntp_servers:
+- corp.intel.com
```
Comment (pbchekin) [inline icl-cluster-profiles/profiles/ci.yaml:39]: Let's move specific configuration to the runner.

File: scripts/ci/init.sh [hunk 4]
```diff
@@ -30,6 +31,10 @@ set_env() {
  
     echo "WORKFLOW_DIR: $WORKFLOW_DIR"
     echo "X1_K8S_EXTRA_SETTINGS_FILE: $X1_K8S_EXTRA_SETTINGS_FILE"
+
+    export LC_ALL="C.UTF-8"
+    export LC_ALL
```
Comment (pbchekin) [inline scripts/ci/init.sh:36]: Why do we need the second export?

File: .github/workflows/deploy-vagrant.yaml (hunk 1 above)
Comment (leshikus) [reply .github/workflows/deploy-vagrant.yaml:32]: Fixed

File: icl-cluster-profiles/profiles/ci.yaml (hunk 3 above)
Comment (leshikus) [reply icl-cluster-profiles/profiles/ci.yaml:39]: Fixed

File: scripts/ci/init.sh (hunk 4 above)
Comment (leshikus) [reply scripts/ci/init.sh:36]: Fixed

File: icl-cluster-profiles/profiles/ci.yaml (hunk 2 above)
Comment (leshikus) [reply icl-cluster-profiles/profiles/ci.yaml:12]: Fixed

Comment (leshikus) [reply icl-cluster-profiles/profiles/ci.yaml:12]: Sorry, another attempt

**Comment by leshikus** [comment]
There is a problem that `virsh --all` somehow does not work after the session is over; probably have to mount some other volume for the host for it to work

**Comment by leshikus** [comment]
Here is a test result, https://github.com/aregm/icl/actions/runs/9915590474/job/27396680921

File: icl-cluster-profiles/profiles/ci.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  {{ ansible_default_ipv4.address }} prefect.{{ ingress_domain }}
+
+  {{ ansible_default_ipv4.address }} ray.{{ ingress_domain }}
+
+  {{ ansible_default_ipv4.address }} registry.{{ ingress_domain }}
+
+  {{ ansible_default_ipv4.address }} s3.{{ ingress_domain }}
+
+  '
+docker_image_repo: dockerhubcache.caas.intel.com
```
Comment (pbchekin) [inline icl-cluster-profiles/profiles/ci.yaml:23]: Please move this to the runner, ideally the whole profile

**Review by pbchekin — APPROVED** [review]

**Pull request merged**